### PR TITLE
[Backend] Limiting the Number of upvotes/downvotes a particular user can do Fixed

### LINKED ISSUE #919: [Backend] Limiting the Number of upvotes/downvotes a particular user can do
**Describe the bug**
As the `Q&A` section is open, users can do the `upvote` and `downvotes` any umber of times.

**To Reproduce**
1. Go to Q&A section of the website - https://hitk-tech-community.netlify.app/q&a 
2. Click on the upvote/downvote button

**Expected behavior**
I am all hears for the perfect solution but i am thinking that there should be a limit either based on the ip of the user of session, user shouldn't be allowed to do more than one upvote/downvote from a single ip/session.

**Screenshots**
Screenshot 2024-05-18 at 11 45 54 AM

### PR DESCRIPTION
**Pull request by Hemu21**
## Issue that this pull request solves
[Issue Link](https://github.com/HITK-TECH-Community/Community-Website/issues/919) resolve #919 
 Closes: #919 

### Brief description of what is fixed or changed
As the Q&A section is open, users can do the upvote and downvotes any umber of times. Now tey can vote only once.

## Types of changes

_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update (Documentation content changed)
- [ ] Other (please describe): 

## Checklist

_Put an `x` in the boxes that apply_

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] My changes does not break the current system and it passes all the current test cases.

### CODE REVIEW

File: backend/app/routes/Q&A/question/upvoteQuestion.js [hunk 1]
```diff
@@ -12,10 +13,11 @@ module.exports = async (req, res, next) => {
       message: 'Database Error',
       errStack: err,
     });
-
     return next(error);
   }
 
+  res.cookie(getVoteCookieName('question', questionId), true, { maxAge: 20 * 365 * 24 * 60 * 60 * 1000 });
```
Comment (Kajol-Kumari) [inline backend/app/routes/Q&A/question/upvoteQuestion.js:19]: what is the unit of this?

**Review by Kajol-Kumari — COMMENTED** [review]
@Hemu21  did u test this change?

File: backend/app/routes/Q&A/question/upvoteQuestion.js (hunk 1 above)
Comment (Hemu21) [reply backend/app/routes/Q&A/question/upvoteQuestion.js:19]: I kept it to 20 years. why because no one can use a mobile for 20 years.

**Comment by Hemu21** [comment]
@Kajol-Kumari  I had tested Everything works fine. see this

https://github.com/HITK-TECH-Community/Community-Website/assets/106808387/5f5cd179-abfb-49c8-acbd-3f6ff44db9fb

**Review by Kajol-Kumari — APPROVED** [review]
LGTM 👍

**Pull request merged**